pkgs/build-support/cc-wrapper/add-clang-cc-cflags-before.sh

```diff
@@ -1,15 +1,24 @@
 needsTarget=true
+targetValue=""
 
 for p in "${params[@]}"; do
     case "$p" in
-    -target | --target=*)
+    -target)
         needsTarget=false
-
-        echo "Warning: supplying the --target argument to a nix-wrapped compiler may not work correctly - cc-wrapper is currently not designed with multi-target compilers in mind. You may want to use an un-wrapped compiler instead." >&2
+        targetValue=$2
+        shift 2
```

Comment:
Shouldn’t this be `shift 1` to avoid an off‐by‐one from the loop advancing? (Does `shift` even work properly with `for`?)

Reply:
That's a good catch. I'm using "${params[@]}" copy anyways, I shouldn't be using positional parameters in there. I'll fix that.